dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -487,19 +527,25 @@ public void Test_Protected_Hidden_Secret() {
                         .getEntity();
                 Assert.assertNotNull(appDetailedView.getSites());
                 Assert.assertFalse(appDetailedView.getSites().isEmpty());
-                final Map<String, Secret> secrets = appDetailedView.getSites().get(0)
-                        .getSecrets();
-                for (Entry<String, Secret> secretEntry : secrets.entrySet()) {
+                final Map<String, SecretView> secrets = appDetailedView.getSites().get(0)
+                        .getSecrets().stream().collect(Collectors.toMap(SecretView::getName,
+                                Function.identity(),(v1, v2) -> v1, LinkedHashMap::new));
+                //Using a LinkedHashMap we guarantee we keep the original order on which the elements were sent.
+                int index = 0;
+                for (Entry<String, SecretView> secretEntry : secrets.entrySet()) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'secretEntry' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5289044)